### DISCUSSION TITLE
v2.8.1

### DISCUSSION THREAD

**Discussion by iBotPeaches** [iBotPeaches/Apktool#3194] [Releases]
## What's Changed
* feat: Upgrade gradle to v8.2.1 by @ArjunaKumarMohanta in https://github.com/iBotPeaches/Apktool/pull/3141
* build(deps): bump gradle/gradle-build-action from 2.5.1 to 2.6.0 by @dependabot in https://github.com/iBotPeaches/Apktool/pull/3151
* fix: skip files if cleaned to empty filename by @iBotPeaches in https://github.com/iBotPeaches/Apktool/pull/3166
* fix: opt out of stricter java rules for zip64 and dot entries by @iBotPeaches in https://github.com/iBotPeaches/Apktool/pull/3178
* build: test on java20 by @iBotPeaches in https://github.com/iBotPeaches/Apktool/pull/3177
* Correct regressions towards apktool.yml generation by @iBotPeaches in https://github.com/iBotPeaches/Apktool/pull/3172
* Allows run app from gradle for debugging purposes by @sv99 in https://github.com/iBotPeaches/Apktool/pull/3179
* fix: add missing hasResources check by @IgorEisberg in https://github.com/iBotPeaches/Apktool/pull/3182
* Prevent loading from resource pool if type is not a resolveable resource by @iBotPeaches in https://github.com/iBotPeaches/Apktool/pull/3187
* style: normalize command descs and redundancy by @IgorEisberg in https://github.com/iBotPeaches/Apktool/pull/3188
* fix: crash when decompiling framework-res.apk with a tag (regression) by @IgorEisberg in https://github.com/iBotPeaches/Apktool/pull/3193

**Full Changelog**: https://github.com/iBotPeaches/Apktool/compare/v2.8.0...v2.8.1
**Blog of Release**: https://connortumbleson.com/2023/07/22/apktool-v2-8-1-released/
**Doc Site**: https://apktool.org/blog/apktool-2.8.1

<hr /><em>This discussion was created from the release <a href='https://github.com/iBotPeaches/Apktool/releases/tag/v2.8.1'>v2.8.1</a>.</em>

**Comment by antoniocarlosdeoliveira** [comment]
Hey how's it going? I was using version 2.7.0 of ApkTools and everything was going fine when compiling the apk, but when I switched to version 2.8.1 when I compile the APK it cannot be installed on TV Box Devices with Android <= 11

So I went back to using version 2.7.0 and everything went well again this is just feedback.

Thanks.

**Reply by iBotPeaches** [reply]
I'm pretty sure we've already patched this. It was an unfortunate regression, but if you see `sparseResources` in your `apktool.yml` between 2.7.0 and 2.8.1. If the value was false on 2.7.0 and true on 2.8. 1 its this ticket: https://github.com/iBotPeaches/Apktool/issues/3199

**Comment by Fingers310** [comment]
Can i use in android?

**Reply by iBotPeaches** [reply]
Not officially. Sorry.

**Comment by saleemjavith** [comment]
unable to update from 2.6 to 2.8 in Kali Linux, tried removing Apktool but unable to do it also. pls help

**Reply by iBotPeaches** [reply]
Kali packages Apktool itself, so not sure what problem you would have. Theres no details to go on here.
